KS.Fiks.IO.Client/Amqp/AmqpReceiveConsumer.cs

```diff
@@ -35,29 +35,11 @@ internal class AmqpReceiveConsumer : DefaultBasicConsumer, IAmqpReceiveConsumer
             Guid accountId)
             : base(model)
         {
-            _dokumentlagerHandler = dokumentlagerHandler;
-            _fileWriter = fileWriter;
-            _decrypter = decrypter;
-            _sendHandler = sendHandler;
-            _accountId = accountId;
-        }
-
-        private static bool IsDataInDokumentlager(IBasicProperties properties)
-        {
-            return ReceivedMessageParser.GetGuidFromHeader(properties.Headers, DokumentlagerHeaderName) != null;
-        }
-
-        private static Guid GetDokumentlagerId(IBasicProperties properties)
-        {
-            try
-            {
-                return ReceivedMessageParser.RequireGuidFromHeader(properties.Headers, DokumentlagerHeaderName);
-            }
-            catch (Exception ex)
-            {
-                Console.WriteLine(ex.Message);
-                throw;
-            }
+            this._dokumentlagerHandler = dokumentlagerHandler;
+            this._fileWriter = fileWriter;
```

Comment:
burde det enten være `_fileWriter` eller `this.fileWriter` ikke en blanding? Antar `this` er preferansen, siden det er blitt endret.

Reply:
Fjernet _